src/saveToCase.js

```diff
@@ -40,6 +42,10 @@ define([
         return mug ? mug.p.use_index : false;
     }
 
+    function attachmentCase(mug) {
+        return mug ? mug.p.use_attachment : false;
```

Comment:
underscore_naming? ... ah, do I spot the precedent four lines above? how did that sneak in there?